src/main/java/com/conveyal/r5/analyst/broker/BrokerHttpHandler.java

```diff
@@ -65,6 +66,10 @@ public void service(Request request, Response response) throws Exception {
         if (pathComponents.length < 2) {
             response.setStatus(HttpStatus.BAD_REQUEST_400);
             response.setDetailMessage("path should have at least one part");
+            TaskError error = new TaskError(null, "Path should have at least one part", "");
```

Comment:
This would usually be caused by an error in how our components communicate with each other, so I'm not sure we need to wrap the error message in a data model object that's intended to show user-oriented errors.